cli/pack/pack.go

```diff
@@ -244,6 +251,54 @@ func setSDKPath(ctx *context.Ctx) error {
 	return nil
 }
 
+func fillScript(filename string, defaultScriptFilePath string, outputScript *string) error {
```

Comment:
I don't gey why we have to return outputScript not explicitly, but with through a pointer